wallet/types/src/keys.rs

```diff
@@ -118,40 +119,13 @@ impl KeychainUsageState {
     }
 }
 
-/// The key id is described by it's public key
-#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
-pub struct RootKeyId(ExtendedPublicKey);
+#[derive(PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
+pub struct RootKeyConstant;
```

Comment:
Please add documentation explaining why this exists and how it works

Reply:
added a comment